script.js

```diff
@@ -1,50 +1,71 @@
-function Animal(name){
+
+class Animal {
+  constructor(name) {
     this.name = name;
-}
+  }
 
-Animal.prototype.action = function(){
-  document.getElementById(this.actionSoundName).play();
+  getImage() {
+    document.getElementById("animalImage").src = this.url;
+
+  }
   
+action() {
+  document.getElementById(this.actionSoundName).play();
```

Comment:
buraya bir `e.stopPropagation();` eklenmesi gerek. 
`  action(e) {
        e.stopPropagation();
        document.getElementById(this.actionSoundName).play();
    }`
Bu şekilde event bubbling'i engelleyerek tr'in onClick eventini tetiklemesinin önüne geçebiliriz.